app/assets/stylesheets/action_page.scss

```diff
@@ -1254,3 +1254,84 @@ html.js #affiliations {
     }
   }
 }
+
+#congress-message-create {
+  position: relative;
+
+  #customize-message .notice {
+    overflow: hidden;
+    max-height: 25em;
+    transition: max-height 250ms linear, padding 250ms ease-in;
+
+    position: absolute;
+    bottom: 0;
+    padding: 1rem 2.5rem;
+    margin: 0 -.8rem -1rem;
+
+    @media (min-width: $md) {
+      margin-left: -2.3rem;
+    }
+
+    background-color: #e5e5e5;
+
+    font-size: .8em;
+
+    &.down {
+      padding-top: 0em;
+      padding-bottom: 0em;
+      max-height: 0em;
+    }
+  }
+
+  #customize-message .notice strong {
+    font-size: 1.2em;
+  }
+
+  #customize-message .notice p {
+    margin-top: 0.7em;
+  }
+
+  #customize-message .notice .actions {
+    display: flex;
+  }
+
+  #customize-message .notice input {
+    flex: 1;
+    padding: 0.4em;
+    margin: 0;
+    border: 0;
+
+    font-size: 1em;
```

Comment:
I see that you are using a combination of ems and rems. We normally try to stick to rems.

Reply:
I find ems very useful. Here I need the font size inside the button input to be the same as it is in the parent container. `1em` seems to be the simplest way to do that. Elsewhere I've used `1.2em` to make the `<strong>` font slightly larger than the surrounding text, which is more flexible than hardcoding the container at `0.8rem` and then using `0.96rem` for the strong font size.

Is there a downside to using `em` in cases like this?

Reply:
Just a matter of consistency, but I sometimes use pixels too in the way you are using ems. Perhaps we could have a group discussion on this at a team meeting.